libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -734,13 +816,20 @@ export class PackageExtractor {
             return;
           }
 
+          // Filter out files that are excluded by the project configuration or dependency configuration.
+          if (
+            relativeSourcePath !== '' &&
```

Comment:
Both of these depend on the relative source path, Just put them under the same if block